### PR TITLE
CIOEC Mandate presentation draft

### PR DESCRIPTION
**Pull request by remyb2canada**
Relates to issue #335 

first draft

### CODE REVIEW

**Comment by smellems** [comment]
This should probably replace the current mandate presentation.
https://github.com/sara-sabr/ITStrategy/blob/master/presentations/en/Mandate.md

**Comment by remyb2canada** [comment]
@smellems reason I didn't replaced current mandate presentation (https://github.com/sara-sabr/ITStrategy/blob/master/presentations/en/Mandate.md) was because CIOEC asks for a specific template and format.
E.g. the first slide needs to be "purpose"

I'm thinking we should have a mandate presentation that is generic as well as having one for a specific committee?

**Comment by gcharest** [comment]
@remyb2canada test fails because the link to O'Reilly is slow to respond. But the URL works so we can actually merge (Guardrails: if exception, validate manually :wink: )

**Comment by gcharest** [comment]
Reran the CI tests and it's now good to go!

File: presentations/en/SLF.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Team's current priorities
+
+- Be the catalysts of change within IITB to lay the path for the evolution of ESDC's digital services.
+
+---
+
+### What's in it for you
+
+- asdfasdfasdf
```
Comment (jaysonmc) [inline presentations/en/SLF.md]: ? @remyb2canada

Comment (remyb2canada) [reply presentations/en/SLF.md]: the SLF.md file was accidently added as part of the PR.
It wasn't supposed to be

I need to stop using
`git add .`

:P

**Comment by remyb2canada** [comment]
> 
> 
> #365

@jaysonmc , please accept the changes so the PR can merge

**Pull request merged**